react/components/Loader.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+const Loader = () => (
```

Comment:
I'd use `Spinner` directly

Reply:
I agree that we need a better loader, but I'll leave that for another PR.